app/services/metrics/send_metrics.rb

```diff
@@ -0,0 +1,26 @@
+module Metrics
+  class SendMetrics
+    PROMETHEUS_THREAD_SLEEP = 0.1
+
+    def self.call
+      new.call
+    end
+
+    def call
+      Metrics::SidekiqQueueSizes.call(prometheus_client)
+
+      # prometheus_exporter runs in a loop and we need to give it some time to send the metrics
+      # https://github.com/discourse/prometheus_exporter/blob/master/lib/prometheus_exporter/client.rb#L54
+      sleep PROMETHEUS_THREAD_SLEEP * 2
```

Comment:
This is because this code is run by a rake task. And I had the issue where the metrics were not sent to prometheus because the rake task would terminate before the prometheus_client had time to send the metrics.
Maybe I should move this `sleep` to the rake task itself then.

Reply:
The spec would be simpler - because it wouldn't need to mock the constant defining the delay.

Reply:
On the other hand, I do want the tests to test that there is a `sleep` at the end